### ISSUE TITLE
2 UMD builds from this version, loaded on the same page, does not work in IE11

### ISSUE THREAD

**Issue by randyhoulahan** [vuejs/vue-cli#5550]
### Version
4.4.1

### Reproduction link
[https://aa.cbddev.xyz/ie/loads-same-monorepo.html](https://aa.cbddev.xyz/ie/loads-same-monorepo.html)

### Environment info
```
  System:
    OS: macOS 10.15.5
    CPU: (12) x64 Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
  Binaries:
    Node: 14.0.0 - /usr/local/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.14.4 - /usr/local/bin/npm
  Browsers:
    Chrome: 83.0.4103.61
    Edge: 83.0.478.44
    Firefox: 75.0
    Safari: 13.1.1
  npmPackages:
    @vue/cli-plugin-babel: ~4.3.0 => 4.3.1 
    @vue/cli-plugin-e2e-cypress: ~4.3.0 => 4.3.1 
    @vue/cli-plugin-eslint: ~4.3.0 => 4.3.1 
    @vue/cli-service: ~4.3.0 => 4.3.1 
  npmGlobalPackages:
    @vue/cli: 4.4.1

browserslist:
[legacy]
last 2 versions
ie 10-11

[modern]
last 2 Chrome versions
last 2 ChromeAndroid versions

last 2 Safari versions
last 2 iOS versions

last 2 Firefox versions
last 2 FirefoxAndroid versions

last 2 Edge versions

[ssr]
maintained node versions


```

### Steps to reproduce
All builds with BROWSERSLIST_ENV=legacy

1. build target lib format umd, umd-min lib one 
2. build target lib format umd, umd-min lib two
3. load both via script tag

https://aa.cbddev.xyz/ie/loads-same-monorepo.html
https://aa.cbddev.xyz/ie/loads-same-monorepo-lib-one.html
https://aa.cbddev.xyz/ie/loads-same-monorepo-lib-two.html

### What is expected?
That there be no webpack conflicts and each umd should load and be present on the window object in IE11.

### What is actually happening?
error: Out of stack space - not clear where from IE
first umd loads second does not.

---
Each used alone is ok, however, together it results in stack overflow. If one was built with vue cli 3 again ok.  Everything is fine in other browsers.

I tried to add output.uniqueName to webpack config for each but 'uniqueName' was not permitted on build error was given on vue-cli-service build command.

**Comment by sodatea** [comment]
Working on that. There's a bug in this dependency: https://github.com/sodatea/get-current-script

I've figured out the fix but needs more testing. If the tests passed I'll release a new version in hours.

**Comment by sodatea** [comment]
Fixed in `@soda/get-current-script` v1.0.1
